test/cli_crud_test.go

```diff
@@ -83,97 +66,3 @@ func TestSimpleFnTriggerUpdateCycle(t *testing.T) {
 	h.Fn("update", "trigger", appName1, funcName1, "mytrigger", "--annotation", `"val1='["val2"]'"`).AssertSuccess()
 	h.Fn("config", "trigger", appName1, funcName1, "mytrigger", "confA", "valB").AssertSuccess()
 }
-func TestRemovingRouteAnnotation(t *testing.T) {
```

Comment:
We should probably migrate this test instead of deleting it. We don't test annotation removal on any other object.